owner/src/main/java/org/the_chance/honeymart/ui/features/signup/SignupScreen.kt

```diff
@@ -81,28 +83,28 @@ fun SignupContent(
                 iconPainter = painterResource(org.the_chance.design_system.R.drawable.ic_password),
                 onValueChange = listener::onConfirmPasswordChanged,
                 errorMessage = state.confirmPasswordState.errorState,
+                keyboardOptions = KeyboardOptions.Default.copy(imeAction = ImeAction.Done),
             )
         }
 
         Column(
             verticalArrangement = Arrangement.SpaceBetween
         ) {
             HoneyFilledButton(
-                label = stringResource(R.string.continue_word),
+                label = stringResource(org.the_chance.design_system.R.string.continue_word),
```

Comment:
Why change import can fix when set with import place or set with named is use another R